src/editor/utils/exportMarkdown.js

```diff
@@ -1,5 +1,6 @@
 /**
- * Before you edit or update codes in this file, make sure you have read the
+ * Hello contributors, Before you edit or update codes in this file,
```

Comment:
I would suggest to use `!` or `.` after contributors.

Reply:
ok